runtime/filetype.lua

```diff
@@ -13,7 +13,13 @@ vim.api.nvim_create_autocmd({ 'BufRead', 'BufNewFile' }, {
   group = 'filetypedetect',
   callback = function(args)
     local ft, on_detect = vim.filetype.match({ buf = args.buf })
-    if ft then
+    if not ft then
+      -- Generic configuration file used as fallback
+      ft = require('vim.filetype.detect').conf(vim.api.nvim_buf_get_name(args.buf), args.buf)
+      if ft then
+        vim.cmd('setf FALLBACK ' .. ft)
```

Comment:
```suggestion
        vim.api.nvim_cmd({ cmd = 'setf', args = { 'FALLBACK', ft }, {})
```
more verbose but structured :)